lib/cn/csched_sp3.c

```diff
@@ -2756,12 +2748,11 @@ sp3_create(
 
     memset(sp, 0, alloc_sz);
     sp->rp = rp;
+    sp->sp_healthy = true;
```

Comment:
This was dropped in a bad rebase on my REST PR onto Greg's space amp work. Prevented node splitting from actually occurring.

Reply:
Lemme see if I can work up a split/join smoke test...

Reply:
So, master seems rather broken right now due to three bugs?  (these two and the other you have a PR up for?)
You should probably send out a note...
I'm trying to use gsr/zspill, but I had to fix some compiler warnings, but it dropped core early into a kmt load....

Reply:
Maybe piggyback these fixes into your current PR and get them in ASAP?